runners/direct-java/src/main/java/org/apache/beam/runners/direct/TransformExecutor.java

```diff
@@ -111,6 +111,7 @@ public void run() {
       TransformEvaluator<T> evaluator =
           evaluatorFactory.forApplication(transform, inputBundle, evaluationContext);
       if (evaluator == null) {
+        onComplete.handleEmpty(transform, inputBundle);
```

Comment:
So this is called only when the evaluator factory returns null? The name might not be perfect here. What is it that is empty?